[2025-04-05T04:55:50.458+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T04:55:50.485+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e_commerce_dw_etl.load.transform_fact_race_results manual__2025-04-05T04:55:45.859805+00:00 [queued]>
[2025-04-05T04:55:50.493+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e_commerce_dw_etl.load.transform_fact_race_results manual__2025-04-05T04:55:45.859805+00:00 [queued]>
[2025-04-05T04:55:50.493+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-04-05T04:55:50.503+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load.transform_fact_race_results> on 2025-04-05 04:55:45.859805+00:00
[2025-04-05T04:55:50.511+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=776) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T04:55:50.512+0000] {standard_task_runner.py:63} INFO - Started process 778 to run task
[2025-04-05T04:55:50.511+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'e_commerce_dw_etl', 'load.transform_fact_race_results', 'manual__2025-04-05T04:55:45.859805+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/ETL_dag.py', '--cfg-path', '/tmp/tmpdshx6ib7']
[2025-04-05T04:55:50.513+0000] {standard_task_runner.py:91} INFO - Job 72: Subtask load.transform_fact_race_results
[2025-04-05T04:55:50.549+0000] {task_command.py:426} INFO - Running <TaskInstance: e_commerce_dw_etl.load.transform_fact_race_results manual__2025-04-05T04:55:45.859805+00:00 [running]> on host 4766eee3ca7d
[2025-04-05T04:55:50.615+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='e_commerce_dw_etl' AIRFLOW_CTX_TASK_ID='load.transform_fact_race_results' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T04:55:45.859805+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-05T04:55:45.859805+00:00'
[2025-04-05T04:55:50.616+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T04:55:50.708+0000] {logging_mixin.py:188} INFO - Các cột trong DataFrame sau khi merge:
[2025-04-05T04:55:50.709+0000] {logging_mixin.py:188} INFO - Index(['resultId', 'raceId', 'driverId', 'constructorId', 'number', 'grid',
       'position', 'positionText', 'positionOrder', 'points', 'laps', 'time_x',
       'milliseconds', 'fastestLap', 'rank', 'fastestLapTime',
       'fastestLapSpeed', 'statusId', 'year', 'round', 'circuitId', 'name',
       'date', 'time_y', 'url', 'fp1_date', 'fp1_time', 'fp2_date', 'fp2_time',
       'fp3_date', 'fp3_time', 'quali_date', 'quali_time', 'sprint_date',
       'sprint_time'],
      dtype='object')
[2025-04-05T04:55:50.792+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-05T04:55:50.813+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-05T04:55:59.563+0000] {logging_mixin.py:188} INFO - Đã transform và lưu dữ liệu vào fact_race_results
[2025-04-05T04:55:59.571+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T04:55:59.573+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T04:55:59.581+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=e_commerce_dw_etl, task_id=load.transform_fact_race_results, run_id=manual__2025-04-05T04:55:45.859805+00:00, execution_date=20250405T045545, start_date=20250405T045550, end_date=20250405T045559
[2025-04-05T04:55:59.623+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T04:55:59.639+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-05T04:55:59.640+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
